src/Components/SuperAdmins/super-admins.module.css

```diff
@@ -93,7 +128,7 @@
   background-color: transparent;
   background-image: url(../../../public/assets/images/super-admin-delete-icon.png);
   background-size: cover;
-  cursor:pointer;
+  cursor: pointer;
 }
 
 .editBtn {
```

Comment:
Same here

Reply:
They were left over from the original css and I forgot to remove them. They are currently not used.